README.md

```diff
@@ -291,22 +291,48 @@ Recipes support out of the box every model, metric or loss that is implemented i
  </br></br>
 
 
-### Using DDP
+### Using Distributed Data Parallel (DDP)
+#### Why use DDP ?
+
+Recent Deep Learning models are growing larger and larger to an extent that training on a single GPU can take weeks.
+In order to train models in a timely fashion, it is necessary to train them with multiple GPUs.
+Using 100s GPUs can reduce training time of a model from a week to less than an hour.
+
+#### How does it work ?
+Each GPU has its own process, which controls a copy of the model and which loads its own mini-batch from disk and sends
+it to its GPU during training. After the forward pass is completed on every GPU, the gradient is reduced across all
+GPUs, yielding to all the GPUs having the same gradient locally. This leads to the model weights to stay synchronized
+across all GPUs after the backward pass.
+
+#### How to use it ?
+You can use SuperGradients to train your model with DDP in just a few lines.
+
 ```python
-from super_gradients import init_trainer
+from super_gradients import init_trainer, Trainer
 from super_gradients.common import MultiGPUMode
 from super_gradients.training.utils.distributed_training_utils import setup_gpu_mode
 
 # Initialize the environment
 init_trainer()
 
-# Launch DDP on 1 device (node) of 4 GPU's
+# Launch DDP on 4 GPUs'
 setup_gpu_mode(gpu_mode=MultiGPUMode.DISTRIBUTED_DATA_PARALLEL, num_gpus=4)
 
-# Define the objects
+# Call the trainer
+Trainer(multi_gpu=MultiGPUMode.DISTRIBUTED_DATA_PARALLEL, expriment_name=...)
+
+# Everything you do below will run on 4 gpus
+
+...
+
+Trainer.train(...)
+
```

Comment:
Maybe add a comment on how to run this (i.e that this can be simply ran as any other python .py file for torch>=XXX and using torch.distributed.launch .. otherwise)?

Reply:
I added : 
Please note that if you work with `torch<1.9.0` (deprecated), you will have to launch your training with either 
`torch.distributed.launch` or `torchrun`, in which case `nproc_per_node` will overwrite the value  set with `gpu_mode`:
```bash
python -m torch.distributed.launch --nproc_per_node=4 main.py
```
```bash
torchrun --nproc_per_node=4 main.py
```